src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+          よくある問題を早期に、そして頻繁にキャッチしてくれることが素晴らしいです。
+          高い拡張性を持つESLintは、チームが自分たちに合ったカスタムルールを実施できるため、
+          オープンソースプロジェクトなどでも優れた効果を発揮しています。
+          ChromeはESLintのスポンサーとして、ウェブ開発者が一貫して高品質のコードを書けるよう支援できることを嬉しく思っています。
+          これは、現代のウェブ開発ツールキットに不可欠な要素です。
+      - name: Jeff Cross
+        title: Co-founder and Principal Architect, Nx
+        image: jeff-cross.jpg
+        text: >
+          NxはESLintをコード品質、ライブラリ境界、およびプロジェクトの可視性を強制する為に使用してます。
+          ESLintはNxをより強力にするので、Nx.devコミュニティを代表して、ESLintの継続的な成功のためにスポンサーになることを嬉しく思います。
+      - name: Médédé Raymond KPATCHAA
+        title: Technical Architect, Salesforce
+        image: medede-kpatchaa.jpg
+        text: >
+          ESLintは、どんなJavaScriptプロジェクトにも使える非常に便利なツールです。
+          このツールを使うとプロジェクトの初期段階から一貫したコードフォーマットルールを設定でき、
+          コードレビューが非常に容易になり、また、新しい開発者がチームに簡単に溶け込めるようなります。
+          私を含めて私多くの開発者からESLintの柔軟性と使いやすさが好まれてます。
+      - name: Duane O'Brien
+        title: Head of Open Source, Indeed
+        image: duane-obrien.jpg
+        text: >
+          ほとんどのJavaScriptプロジェクトは、直接または間接的にESLintに依存しています。
+          Indeedの社員がFOSS Contributor Fundから寄付を受けるためにESLintを推薦し、その同僚も明らかに同意しています。
+          オープンソースの技術は我々の製品を支え、人々の就職を支援するデジタルインフラを開発し、保護し、維持することを可能にするものです。
+          そのオープンソース技術を支援できることを嬉しく思っています。
+      - name: Gajus Kuizinas
+        title: Co-Founder / CTO, Contra
+        image: gajus.jpg
+        text: >
+          ESLintは、増え続けるコードベースとエンジニアに安全で一貫した体験を提供するために大きな役割を担っています。
+          私たちは、Contraの構築を助けてくれているすべてのオープンソースプロジェクトに感謝していますし、
+          ESLintは、私たちがサポートすることを約束する多くのオープンソースプロジェクトの最初のものです。
+
+  faq:
+    title: よくあるご質問
+    description: 製品・課金に関するすべての情報. お探しの答えが見つかりませんか？私たちのフレンドリーなチームとチャットでもできます。
+    items:
+      - q: 寄付されたお金は誰が受け取るのですか？
+        a:
+          <a href='https://opencollective.com'>Open Collective</a>またはGitHubを通じての寄付であろうと、
+          すべての寄付は、501(c)(3)の非営利団体である。Open Source Collectiveによって受け取られます。
+          The Open Source CollectiveはESLintの財政的なホストとして、すべての寄付を記録しています。
+      - q: 寄付の領収書がもらえますか?
+        a:
+          OpenCollectiveで寄付をされた場合、PDFの領収書をお送りします。GitHubで寄付をされた場合は、
+          領収書または請求書に記載されます。
+      - q: 寄付金の税額控除はできますか?
+        a:
+          オープンソースコレクティブは501(c)(3)非営利団体ですが、IRSは、オープンソースソフトウェアの開発を慈善事業と
+          見なしていません。そのため、非課税措置の対象となりません。
+      - q: お金の分配は誰が決めるのですか？
+        a: TSCが、誰がどれだけお金を受け取るかをすべて決定します。
+      - q: お金の使い道を確認することはできますか？
+        a:
+          できます。Open Collectiveのページでは、申請・決済されたすべての経費を確認することができます。
+          すべての取引は完全にオープンで、一般に公開されています。
+      - q: 定期的な寄付をいつでもキャンセルできますか？
+        a:
+          できます。長期的なコミットメントはありません。寄付の取り消しはいつでも可能です。Open CollectiveまたはGitHubを
+          ログインしてからキャンセルができます。
+      - q: ロゴがホームページ、GitHub、npmパッケージに表示されるまでには、どれくらいの時間がかかりますか？
+        a:
+          ホームページとGitHubのロゴは毎日自動的に更新されますので、24時間以内に表示されるはずです。
+          npmパッケージのロゴは、通常2週間ごとに行われる新しいリリースの公開時にのみ更新されます。
+          ただし、メジャーリリースは数ヶ月かかることが多いので、その間はnpmパッケージのロゴは更新されません。
+      - q: ロゴとURLはどこから来ているのですか？
```

This may be a more Japanese way of saying it.

```suggestion
      - q: ロゴとURLはどこから取得されますか？
```